yarn.lock

```diff
@@ -13378,11 +13378,16 @@ prettier-linter-helpers@^1.0.0:
   dependencies:
     fast-diff "^1.1.2"
 
-prettier@^2.0.4, prettier@^2.0.5:
+prettier@^2.0.5:
```

Comment:
@iesahin please, make a separate PR for this change so that we fix master

Reply:
I think it's OK now.

Reply:
yep, thanks! can we merge/rebase so that we don't see it here, please?